src/main.c

```diff
@@ -2174,6 +2180,114 @@ void on_middle_click() {
+{
+    char str2[end - start];
+    for (int i = 0; i < (end - start); i++)
+    {
+        str2[i] = str[start + i];
+    }
+    int tru = 1;
+    if (str2[0] != 'C')
+        tru = 0;
+    if (str2[1] != 'R')
+        tru = 0;
+    if (str2[2] != 'A')
+        tru = 0;
+    if (str2[3] != 'F')
+        tru = 0;
+    if (str2[4] != 'T')
+        tru = 0;
+    return tru;
+}
+
+/// As a part of [Issue #41]: https://github.com/Team-10-But-Better/Craft/issues/41
+/// Since keybinds are created as "#defines" in the config.h file, read all lines of the config.h file
+/// and print all lines of which define Craft keybinds.
+void parseConfigToPrintKeybinds()
+{
+    int filedes = open("/home/user/Craft/src/config.h", O_RDONLY);
+    char buffer[2048];
+    read(filedes, buffer, 2048);
+    char* token;
+    const char s[2] = "\n";
+    token = strtok(buffer, s);
+    while (token != NULL)
+    {
+        if (strlen(token) >= 14)
+        {
+            if (isDefinedCraftKey(token, 8, 13) == 1)
+            {
+                printf("%s\n", token);
+            }
+        }
+        token = strtok(NULL, s); 
+    }
+    close(filedes);
+}
+
+/// As a part of [Issue #7]: https://github.com/Team-10-But-Better/Craft/issues/7
+/// Increment the mouse and key DPI by raising the value of the multiplier used to calculate the DPI.
+void incrementDPI()
+{
+    if (mouseDPIMultiplier < .001)
+    {
+        mouseDPIMultiplier += .0001;
+        printf("Mouse DPI Multiplier: %f\n", mouseDPIMultiplier);
+    }
+    else
+    {
+        printf("Cannot increment mouse DPI multiplier any further!\n");
+    }
+    if (keyDPIMultiplier < 2.0)
+    {
+        keyDPIMultiplier += .1;
+        printf("Key DPI Multiplier: %f\n", keyDPIMultiplier);
+    }
+    else
+    {
+        printf("Cannot increment key DPI multiplier any further!\n");
+    }
+}
+
+/// As a part of [Issue #7]: https://github.com/Team-10-But-Better/Craft/issues/7
+/// Decrement the mouse and key DPI by lowering the value of the multiplier used to calculate the DPI.
+void decrementDPI()
+{
+    if (mouseDPIMultiplier >= .0001)
+    {
+        mouseDPIMultiplier -= .0001;
+        printf("Mouse DPI Multiplier: %f\n", mouseDPIMultiplier);
+    }
+    else
+    {
+        printf("Cannot decrement mouse DPI multiplier any further!\n");
+    }
+    if (keyDPIMultiplier > 1)
+    {
+        keyDPIMultiplier -= .1;
+        printf("Key DPI Multiplier: %f\n", keyDPIMultiplier);
+    }
+    else
+    {
+        printf("Cannot decrement key DPI multiplier any further!\n");
+    }
+}
+
+/// As a part of [Issue #7]: https://github.com/Team-10-But-Better/Craft/issues/7
+/// Set the mouse and key DPI by changing the DPI multiplier used to calculate DPI to a predetermined default value.
+void setDefaultDPI()
+{
+    mouseDPIMultiplier = .0001;
+    printf("Mouse DPI Multiplier set to default: .0001\n");
+
+    keyDPIMultiplier = 1.1;
+    printf("Key DPI multiplier set to default: 1.1\n");
```

Comment:
I do not see a good reason to have print statements here in main.c . Reason is if this game were to be shipped, there would be no major benefit to the end user for print statements of which they can not see in game. Such information should really be exclusive to the developer and for testing purposes. Right now all this really does, is make it so more processing by the computer is needed in the background for no major benefit. Thus I recommend your remove said print statement.